mse-run.sh

```diff
@@ -48,8 +49,10 @@ set_default_variables() {
     DRY_RUN=0
     MEMORY=0
     FORCE=0
-    TIMEOUT_DATE=""
-    UUID=""
+    TIMEOUT=""
+    ID=""
+    SUBJECT="CN=cosmian.app,O=Cosmian Tech,C=FR,L=Paris,ST=Ile-de-France"
+    SUBJECT_ALTERNATIVE_NAME="localhost"
```

Comment:
I think `SUBJECT_ALTERNATIVE_NAME` should be the value of `--host` (to remane?) and `HOST` should be `0.0.0.0`